src/compiler/program.ts

```diff
@@ -15,17 +15,23 @@ namespace ts {
     export const version = "1.9.0";
 
     export function findConfigFile(searchPath: string, fileExists: (fileName: string) => boolean): string {
-        let fileName = "tsconfig.json";

```

Comment:
i do not think we should do this. we should not automatically crawl these files. 

the original issue was about `tsc --p jsconfig.json`, and for that, all we need is path an optional argument to `findConfigFile` for the fileName, that defaults to `tsconfig.json` if not specified.